data/xml/2020.wosp.xml

```diff
@@ -0,0 +1,125 @@
+      <url hash="1b34cf2b">2020.wosp-1.8</url>
+    </paper>
+    <paper id="9">
+      <title>2</title>
+      <author><first>Shubhanshu</first><last>Mishra</last></author>
+      <author><first>Sudhanshu</first><last>Mishra</last></author>
+      <pages>59–64</pages>
+      <abstract>We present our team Scubed’s approach in the ‘3C’ Citation Context Classification Task, Subtask A, citation context purpose classification. Our approach relies on text based features transformed via tf-idf features followed by training a variety of models which are capable of capturing non-linear features. Our best model on the leaderboard is a multi-layer perceptron which also performs best during our rerun. Our submission code for replicating experiments is at: https://github.com/napsternxg/Citation_Context_Classification.</abstract>
+      <url hash="17a8f207">2020.wosp-1.9</url>
+    </paper>
+    <paper id="10">
+      <title>2</title>
+      <author><first>Shubhanshu</first><last>Mishra</last></author>
+      <author><first>Sudhanshu</first><last>Mishra</last></author>
+      <pages>65–70</pages>
+      <abstract>We present our team Scubed’s approach in the 3C Citation Context Classification Task, Subtask B, citation context influence classification. Our approach relies on text based features transformed via tf-idf features followed by training a variety of simple models resulting in a strong baseline. Our best model on the leaderboard is a random forest classifier using only the citation context text. A replication of our analysis finds logistic regression and gradient boosted tree classifier to be the best performing model. Our submission code can be found at: https://github.com/napsternxg/Citation_Context_Classification.</abstract>
+      <url hash="466bb438">2020.wosp-1.10</url>
+    </paper>
+    <paper id="11">
+      <title>2</title>
+      <author><first>Premjith</first><last>B</last></author>
+      <author><first>Soman</first><last>Kp</last></author>
```

Comment:
I think this is first initials and last names.

Reply:
Here's what the paper has:

![image](https://user-images.githubusercontent.com/455256/97023580-b8005400-1523-11eb-8487-41a4f4dff95d.png)

Reply:
He does have [an Anthology page](https://www.aclweb.org/anthology/people/s/soman-kp/)